bindings/rust/src/lib.rs

```diff
@@ -1310,4 +1310,31 @@ mod tests {
             "invalid offset or size"
         );
     }
+
+    #[test]
+    fn execute_host_function() {
```

Comment:
Maybe test name should reflect that it tests error condition with missing import.

Reply:
Fixed.